macos/FluentUI/Button/Button.swift

```diff
@@ -228,16 +233,15 @@ open class Button: NSButton {
 		}
 	}
 
-	private var restBackgroundColor: NSColor {
-		return fillColor
-	}
-
+	@objc public var restBackgroundColor: NSColor? = nil
+	@objc public var cornerRadius: CGFloat = 3
```

Comment:
I think we should also have a `needsDisplay = true` to ensure that the view is re-rendered when these change